Makefile

```diff
@@ -29,6 +29,9 @@ uv-installed:
 .PHONY: ${VENV_DIR}/
 ${VENV_DIR}/: uv-installed
 	XDSL_VERSION_OVERRIDE="0+dynamic" uv sync ${VENV_EXTRAS}
+	@if [ ! -z "$(XDSL_MLIR_BIN)" ]; then \
```

Comment:
I would probably rename this `MLIR_OPT_PATH` and point directly to mlir-opt instead?